screenings.go

```diff
@@ -0,0 +1,142 @@
+package checkr
+
+import (
+	"log"
+	"sync"
+)
+
+// Screenings ...
+type Screenings struct {
+	SSNTrace                     *SSNTrace
+	SexOffenderSearch            *SexOffenderSearch
+	GlobalWatchListSearch        *GlobalWatchListSearch
+	NationalCriminalSearch       *NationalCriminalSearch
+	FederalCriminalSearch        *FederalCriminalSearch
+	CountryCriminalSearches      []*CountryCriminalSearch
+	StateCriminalSearch          []*StateCriminalSearch
+	MotorVehicleReport           *MotorVehicleReport
+	EducationVerification        *EducationVerification
+	EmploymentVerification       *EmploymentVerification
+	IdentityDocumentVerification *IdentityDocumentVerification
+}
+
+// GetScreenings returns all the screenings for a report
+func GetScreenings(report *Report, c *Client) *Screenings {
+	cs := &Screenings{}
+	var err error
```

Comment:
This will cause race conditions.

Reply:
agreed i will return an error array

Reply:
I kind of created another Error struct ( ErrorX) for this and added a screening Error which is a map of screen type and the new error i added.  While returning it marshals it as a string and returns.
the Error stuct i created has the ability to store ( response , err , screentype ) which can be used to when we return.